src/Cloner.js

```diff
@@ -253,7 +253,7 @@ class Cloner {
       try {
         // Create the item we will be inserting/updating.
         const destItem = await this.findLast(`dest.${collection}`, this.findQuery(current, find));
-        const updatedItem = _.assign(destItem || {}, _.omit(srcItem, ['_id']));
+        const updatedItem = {...destItem, ...(_.omit(srcItem, ['_id']))};
```

Comment:
@brendanbond I believe we may have `null` for `destItem`, so better like this

```
const updatedItem = {...(destItem || {}), ...(_.omit(srcItem, ['_id']))};

```
I also don't quite like lodash `omit` function as it has quite heavy logic in it and can be slow, so maybe if you'll find a way to omit the property with object destructure or maybe `delete` operator can suite for that case, I think it could make a slight improvement for large db's cloning.
But up to you, not a strict requirement

Reply:
The spread operator on a null (when used in an object) results in an empty object. Try this in a REPL:

```
const obj = null;
const newObj = { ...obj, foo: 'bar' };
console.log(newObj); // logs "{foo: 'bar'}"
```

This is because the spread operator is designed to be equivalent to `Object.assign()`, which when passed null or undefined will coerce to an empty object. A note that this doesn't work equivalently in arrays.

Reply:
@brendanbond didn't know that, good to go then